src/rsa.rs

```diff
@@ -165,10 +167,19 @@ fn parse_public_key<'a>(input: untrusted::Input<'a>) ->
     })
 }
 
+static PREFIXES: [(&'static digest::Algorithm, &'static [u8]); 4] = [
+    (&digest::SHA1, b"\x30\x21\x30\x09\x06\x05\x2b\x0e\x03\x02\x1a\x05\x00\x04\x14"),
```

Comment:
These prefixes are already defined immediately above. See SHA512_PKCS1_DIGESTINFO_PREFIX and friends.